packages/devtools_app/lib/src/service_manager.dart

```diff
@@ -40,12 +40,14 @@ const debugLogServiceProtocolEvents = false;
 
 const defaultRefreshRate = 60.0;
 
+typedef IsolateF = Future<Isolate> Function(String isolateId);
```

Comment:
Dart style: avoid abbreviations.https://dart.dev/guides/language/effective-dart/design#avoid-abbreviations

Reply:
Thanks! This typedef is now gone